modals/src/main/java/com/decathlon/vitamin/compose/modals/VitaminModals.kt

```diff
@@ -84,28 +89,30 @@ object VitaminModals {
         contentScrollState: ScrollState = rememberScrollState(),
         colors: ModalColors = VitaminModalColors.primary(),
         sizes: ModalSizes = VitaminModalSizes.medium(),
-        positiveButton: (@Composable VitaminModalButtons.() -> Unit)? = null,
-        negativeButton: (@Composable VitaminModalButtons.() -> Unit)? = null,
-        neutralButton: (@Composable VitaminModalButtons.() -> Unit)? = null,
+        buttons: List<(@Composable VitaminModalButtons.() -> Unit)> = listOf(),
```

Comment:
We need to update the kdoc if we go for the button list, at least to explain the display order of buttons.
Personally I think it was more clear with positiveButton/negativeButton/neutralButton parameters, especially if there could be only 3 buttons in the end

Reply:
Kdoc & README updated